src/actions/importing/abstract-importing-action.php

```diff
@@ -169,6 +169,13 @@ public function set_completed( $completed ) {
 		$this->options->set( 'importing_completed', $current_importers_completions );
 	}
 
+	/**
+	 * Returns whether the importing action is enabled.
+	 *
+	 * @return bool True if the importing action is enabled.
+	 */
+	abstract public function is_enabled();
```

Comment:
I would implement this function and let it return true by default and override it if any other logic applies. Now we are specifically using it to check for a feature-flag, but if we have multiple importers we would have them enabled by default.